website/static/js/project.js

```diff
@@ -256,6 +256,17 @@ $(document).ready(function() {
         trigger: 'hover'
     });
 
+    var DiscussionsInfoHtml = 'A contributor who is not subscribed to email discussions will ' +
+        'not recieve any of the emails sent to it. They will, however, still be able to send ' +
+        'emails themselves, which will be distributed as per how the discussions list was set.' +
+        '<br><br>You can change your subscription setting on the settings page.';
```

Comment:
The use of `<br>` tags for layout is strongly discourage. Can you wrap both sections in `<p>` tags instead?